Makefile

```diff
@@ -74,10 +74,10 @@ endif
 
 .PHONY: install.tools
 install.tools:
-	$(GO) get -u $(BUILDFLAGS) github.com/cpuguy83/go-md2man
-	$(GO) get -u $(BUILDFLAGS) github.com/vbatts/git-validation
-	$(GO) get -u $(BUILDFLAGS) github.com/onsi/ginkgo/ginkgo
-	$(GO) get -u $(BUILDFLAGS) gopkg.in/alecthomas/gometalinter.v1
+	GO111MODULE=off $(GO) get -u $(BUILDFLAGS) github.com/cpuguy83/go-md2man
+	GO111MODULE=off $(GO) get -u $(BUILDFLAGS) github.com/vbatts/git-validation
+	GO111MODULE=off $(GO) get -u $(BUILDFLAGS) github.com/onsi/ginkgo/ginkgo
+	GO111MODULE=off $(GO) get -u $(BUILDFLAGS) gopkg.in/alecthomas/gometalinter.v1
```

Comment:
I would vendor the tools within the repo to have reproducible outputs from them.

Reply:
Doing that will make it _very_ hard for downstream packagers to figure out which dependency is needed for Buildah and which for it's CI. Debian, for instance, needs to package **all** dependencies in separate .deb packages. I think @siretart will shoot at me :^) Maybe, we can workaround that by creating another module.

Reply:
Yes, that should work. Maybe we can use the same trick for CRI-O to move the CI deps out of the main `go.mod`?

Reply:
I was wondering how this influences Debian packaging. If every (go) dependency is part of the repo then it's not a dependency anymore from the perspective of packaging, right? :grimacing: 

Yes, the idea would be to add a `tools` package, which will never build at all: https://github.com/cri-o/cri-o/blob/master/tools/tools.go 
The go modules discovery via `go list` will add them to `go.mod`.

Reply:
Debian can't use the `./vendor` folder, and all dependencies have to be packaged separately in to a .deb. Mixing build and test dependencies into one module would complicate update a lot as the downstream packagers would have to figure out which is which.